config/src/config/consensus_config.rs

```diff
@@ -245,35 +260,39 @@ impl Default for ConsensusConfig {
                     max_sending_block_txns_override: 10000,
                     max_sending_block_bytes_override: 5 * 1024 * 1024,
                     backoff_proposal_delay_ms: 150,
+                    max_txns_from_block_to_execute: None,
                 },
                 ChainHealthBackoffValues {
-                    backoff_if_below_participating_voting_power_percentage: 77,
+                    backoff_if_below_participating_voting_power_percentage: 78,
                     max_sending_block_txns_override: 2000,
-                    max_sending_block_bytes_override: 1024 * 1024 + BATCH_PADDING_BYTES as u64,
+                    max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
                     backoff_proposal_delay_ms: 300,
+                    max_txns_from_block_to_execute: None,
                 },
                 ChainHealthBackoffValues {
-                    backoff_if_below_participating_voting_power_percentage: 75,
+                    backoff_if_below_participating_voting_power_percentage: 76,
                     max_sending_block_txns_override: 1000,
-                    // stop reducing size, so 1MB transactions can still go through
-                    max_sending_block_bytes_override: 1024 * 1024 + BATCH_PADDING_BYTES as u64,
+                    max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
                     backoff_proposal_delay_ms: 300,
+                    max_txns_from_block_to_execute: None,
                 },
                 ChainHealthBackoffValues {
-                    backoff_if_below_participating_voting_power_percentage: 72,
+                    backoff_if_below_participating_voting_power_percentage: 74,
                     max_sending_block_txns_override: 500,
-                    max_sending_block_bytes_override: 1024 * 1024 + BATCH_PADDING_BYTES as u64,
+                    max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
                     backoff_proposal_delay_ms: 300,
+                    max_txns_from_block_to_execute: None,
                 },
                 ChainHealthBackoffValues {
-                    backoff_if_below_participating_voting_power_percentage: 69,
+                    backoff_if_below_participating_voting_power_percentage: 72,
                     // in practice, latencies and delay make it such that ~2 blocks/s is max,
                     // meaning that most aggressively we limit to ~500 TPS
                     // For transactions that are more expensive than that, we should
                     // instead rely on max gas per block to limit latency
                     max_sending_block_txns_override: 250,
-                    max_sending_block_bytes_override: 1024 * 1024 + BATCH_PADDING_BYTES as u64,
+                    max_sending_block_bytes_override: MIN_BLOCK_BYTES_OVERRIDE,
                     backoff_proposal_delay_ms: 300,
+                    max_txns_from_block_to_execute: None,
```

Comment:
All the max_txns_from_block_to_execute are None in ChainHealthBackoffValues.
Don't we want to consider stricter ChainHealthBackoffValues ?

Reply:
oh. I wrote a separate PR assuming they cannot go in the same release.

but the support you built there is what both use, so they can go. let me just add those changes in this PR...

thanks for the callout :)

Reply:
updated